capture/record.ts

```
@@ -0,0 +1,263 @@
+  })
+
+  // Whatever happens, the session must not outlive its capture: a leaked dev
+  // server holds the fixture's lock and port and poisons every later scenario.
+  try {
+    if (capture.drive) {
+      await capture.drive({ session, cwd, bin: values.bin! })
+    }
+    if (!capture.stopAfterDrive) {
+      let timer: NodeJS.Timeout | undefined
+      const timedOut = new Promise<{ timedOut: true, code: null }>((resolve) => {
+        timer = setTimeout(() => resolve({ timedOut: true, code: null }), 240_000)
```

ESLint

[error] Pass function and arguments directly to timer function to avoid allocating an extra function

(e18e/prefer-timer-args)